packages/foundation-ui/src/mount/__tests__/MountService-test.js

```diff
@@ -20,25 +20,25 @@ describe("MountService", function() {
   });
 
   describe("registerComponent", function() {
-    it("should not throw if a valid React.Component is provided", function() {
+    it("not throw if a valid React.Component is provided", function() {
```

Comment:
The it("not" is located in multiple other tests as well :)

Reply:
-> doesn't